Logging/src/PsrLogger.php

```diff
@@ -449,16 +465,10 @@ public function serialize()
             $this->batchMethod,
             $this->logName,
             $debugOutputResource
-        ]);
+        ];
     }
 
-    /**
-     * Unserializes data.
-     *
-     * @param string
-     * @access private
-     */
-    public function unserialize($data)
+    public function __unserialize(array $data): void
```

Comment:
We cannot have return types because this library is still compatible with PHP 5.5

```suggestion
    public function __unserialize(array $data)
```